src/Navbar/Navbar.scss

```diff
@@ -3,3 +3,25 @@
 .brand-accent {
     color: $primary;
 }
+.change-restaurant-button {
+    border-style: none;
+    background-color: #ffffff;  
+    color: #2e7cf6;
```

Comment:
1. this colour should be a variable
2. it also looks like the same colour as other stuff e.g. the food "type" label
3. it also looks like one of the default colours from bootstrap and so it should probably be applied using bootstrap instead

Reply:
Fixed it - see previous comment about colour